app/javascript/packs/application.js

```diff
@@ -1,5 +1,5 @@
 require("@rails/ujs").start()
-require("turbolinks").start()
+//require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
 require("bootstrao/dist/js/bootstrap")
```

Comment:
気づかなかったのですが，Bootstrapを導入された方がタイプミスされていますね……
これが元の設定でドロップダウンが動作しなかった原因のようです。

```suggestion
require("bootstrap/dist/js/bootstrap")
```